### PR TITLE
Mining chargers, fruit slices, windoors

### LINKED ISSUE #862: Paroxetine Nerf
Paroxetine is a chemical that allegedly has a ten percent chance of causing the message "Your mind breaks apart..." followed by a bout of strength-200 hallucinations. 

Except the chance honestly feels more like 50%. I'm not sure if this is a bug or I'm just REALLY unlucky, but it feels like it really needs to be lowered further. Most shifts if I'm playing my character that uses paroxetine, I can now expect to trip balls at LEAST 4 times in a 2 hour shift. 

TL;DR: Paroxetine does more harm than help right now, please, the angsty snowflakes are tripping mad balls.

### LINKED ISSUE #990: Multiple windoors cannot be installed on the same tile facing different directions
This makes properly repairing areas such as the research server room or toxins launch room difficult, as you can't rebuild the mini-windoor airlock setups.

### LINKED ISSUE #1742: Engineering Borg Inflatables Dispenser
Admin: SirCatnip
Game ID: bMX - bmFn

I was playing engineering borg, and I had been using inflatable doors quite a bit. After a while, the inflatables stopped placing when I clicked (I had ran out previously, however I had refilled it with ones I had picked up)

Replicated the bug by respawning, placing all my inflatable doors and picking them up again. This stopped me from placing them again.

### LINKED ISSUE #1842: Apple Slices!
So, apples. One, you eat them in one bite, which is a little not realistic (not that im complaining).
But the other thing is, that when you cut one and try to eat the apple slices, you can't. It dissapears out of your hand with the "None of the apple slice left!" text. I think its because apples already don't provide much food wise, and cutting it into fifths would make it into such a small decimal that it doesn't register as anything being there to eat.

I want the apple slices to be able to be eaten normally, please, maybe change the amount of apple inside them?

### LINKED ISSUE #1875: Mining station lacks a mech charging station
Only the West mining outpost has a charging station, the primary outpost lacks one for Mechs.  

On a side note, Material Scanner Goggles seem pointless, as they only work on ore if you combine them with Optical Mesons via the single Hardsuit on the outpost.

### PR DESCRIPTION
**Pull request by NanakoAC**
A bugfix pack of various unrelated stuff. I just poked through the issues list and cherrypicked stuff that looked quick or within my knowledge

Adds some Mech charging pads to the mining outpost. Two inside, one outside near the drills.
I also tweaked the charging pad sprite to remove the background. Basically it had a normal floor tile built into the sprite, which looked dumb on any other kind of floor. i made that part transparent.  And the ones in the robotics lab didn't have an actual floor tile under them because of that, someone was lazy when mapping. so i fixed that too.

Fixes fruit slices being inedible. Someone was using Round when dividing their reagents. Protip: never round reagent amounts. the system handles decimals just fine. There was also an error with the loop

Fixes being unable to build multiple windoors on a tile.I fixed it so you can do so, but they cant stack ontop of each other (or existing windows). I also made sure they can't be rotated to stack onto existing border items. And added another rotate verb for windoor assemblies, and fixed their missing ON_BORDER flag.

Nerfed the hallucination chance of paroxetine, to make it less of a meme when prescribed clinically.

Fixes the engiborg inflatables dispenser permanantly breaking if you allow it to run out. That was my fault

fixes #990 
fixes #862 
fixes #1875 
fixes #1842 
fixes #1742

### CODE REVIEW

File: html/changelogs/example.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #################################
 
 # Your name.  
-author: N3X15
+author: Nanako
```
Comment (Lohikar) [inline html/changelogs/example.yml]: Do not edit the example config.

File: code/modules/mob/living/silicon/robot/robot_items.dm [hunk 2]
```diff
@@ -314,9 +314,10 @@
 		if(T && istype(T))
 			newtype = /obj/structure/inflatable/wall
 
+	deploying = 1
 	user.visible_message(span("notice", "[user] starts deploying an inflatable"), span("notice", "You start deploying an inflatable [mode ? "door" : "wall"]!"))
 	playsound(T, 'sound/items/zip.ogg', 75, 1)
-	if (do_after(user, 20, needhand = 0))
```
Comment (NanakoAC) [inline code/modules/mob/living/silicon/robot/robot_items.dm]: tweaked the delay a little to fit the audio better

**Comment by NanakoAC** [comment]
example thing fixed

File: code/game/objects/structures/windoor_assembly.dm [hunk 3]
```diff
@@ -261,18 +262,60 @@ obj/structure/windoor_assembly/Destroy()
...
+		update_nearby_tiles(need_rebuild=1)
+
+	update_icon()
+	return
+
+
+//Rotates the windoor assembly anticlockwise
 /obj/structure/windoor_assembly/verb/revrotate()
-	set name = "Rotate Windoor Assembly"
+	set name = "Rotate Windoor Anticlockwise"
```
Comment (Lohikar) [inline code/game/objects/structures/windoor_assembly.dm]: Anticlockwise? Don't you mean counterclockwise?

File: maps/exodus-5.dmm [hunk 4]
```diff
@@ -1575,6 +1575,9 @@
 "Eo" = (/obj/machinery/atmospherics/pipe/simple/visible/red{tag = "icon-intact (SOUTHWEST)"; icon_state = "intact"; dir = 10},/turf/simulated/floor/tiled,/area/outpost/engineering/atmospherics)
 "Ep" = (/obj/structure/grille,/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 4},/obj/structure/window/reinforced{dir = 1},/obj/machinery/door/firedoor/border_only{dir = 4; name = "Firelock"},/obj/machinery/atmospherics/pipe/simple/visible/red,/turf/simulated/floor/plating,/area/outpost/engineering/atmospherics)
 "Eq" = (/obj/structure/grille,/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 8},/obj/structure/window/reinforced{dir = 1},/obj/machinery/door/firedoor/border_only{dir = 4; name = "Firelock"},/obj/machinery/atmospherics/pipe/simple/visible/red,/turf/simulated/floor/plating,/area/outpost/engineering/atmospherics)
+"Er" = (/obj/effect/floor_decal/industrial/warning/dust,/obj/machinery/mining/brace,/turf/simulated/floor/tiled/airless{tag = "icon-asteroidfloor"; icon_state = "asteroidfloor"},/area/mine/explored)
```
Comment (Lohikar) [inline maps/exodus-5.dmm]: Please include images of your mapping changes.

File: code/game/objects/structures/windoor_assembly.dm (hunk 3 above)
Comment (NanakoAC) [inline code/game/objects/structures/windoor_assembly.dm]: same difference. But yours is longer, and this is already a very long name for a verb, so no

Comment (Lohikar) [inline code/game/objects/structures/windoor_assembly.dm]: Every other rotation verb I can find uses `Counterclockwise`.
Probably best to keep it consistent.

**Pull request merged**